martian-lab/tests/error_test/error_test.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Cool! Add this to travis https://github.com/martian-lang/martian-rust/blob/master/.travis.yml#L15 ?